src/dpo/src/Optdp.cpp

```diff
@@ -549,12 +579,11 @@ void Optdp::createNetwork() {
         ptr->setOffsetY(dy);
         ptr->setPinHeight(hh);
         ptr->setPinWidth(ww);
-
-        // XXX: Not correct, but okay for now!
-        ptr->setPinLayer(0);
+        ptr->setPinLayer(0); // Set to zero since not currently used.
 
         ++p;  // next pin.
       } else {
+        logger_->error(DPO, 103, "Could not find node for instance while connecting pins.");
```

Comment:
It is helpful to mention the object that generated the error as it aids debugging from the logs.  Similarly below.